ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -311,6 +311,7 @@ private void processGetBlobInfoResponse(GetRequestInfo getRequestInfo, GetRespon
     } else {
       logger.trace("Replica {} returned an error {} for a GetBlobInfoRequest with response correlationId : {} ",
           getRequestInfo.replicaId.getDataNodeId(), getError, getResponse.getCorrelationId());
+      processServerError(getError);
```

Comment:
So the old code would only consider partition level error and calling `processServerError` here will ensure that server-level errors (like Disk_Unavailable) will be tracked by `ResponseHandler`?

Reply:
Correct. For instance, if all the servers returned `Disk_Unavailable` or `Unknown_Error`, the current code will not return any response